typescript/packages/lattice-client-core/src/helpers.ts

```diff
@@ -1,79 +1,95 @@
-import {type QueuedIterator} from 'nats.ws';
-import {connect} from 'nats.ws';
+import {type QueuedIterator} from "nats.ws";
+import {connect} from "nats.ws";
 import {
```

Comment:
nit: looks like this is a lot of lint changes (spaces -> tabs, " -> '), should we revert those?

Reply:
You are absolutely right, I used the biome linter/formatter and it automatically adjusted things. It looks better now.